spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -1304,7 +1444,7 @@ the JAX-RS `@Suspended` annotation:
 
 [source,java]
 ----
-    @LRA(value = LRA.Type.REQUIRED, // the method must run with an LRA
+    @LRA(value = LRA.Type.REQUIRES_NEW, // the method must run with an LRA
```

Comment:
REQUIRES_NEW means there is no reason to inject 

```java
@HeaderParam(LRA_HTTP_CONTEXT_HEADER)
```

We use REQUIRED in a lot more examples than REQUIRES_NEW so I removed it in order to:

- balance it up (to show that handling nested contexts isn't always required);
- avoid too much repetition of text around handling nested contexts (since the reader will get the idea after the first few examples).

Reply:
@ochaloup I have just pushed another commit (and squashed) that explains why there are no checks for a nested context when the method is annotated with REQUIRES_NEW (line 1499 below).